appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt

```diff
@@ -101,8 +88,6 @@ class AppInfoPresenter(private val appName: String?, private val repository: Dis
         val memo = "1-${repository.getStoredMemo()}$memoDelim${app?.memo}"
```

Comment:
this should probably not contain the 1-appid since the KinSDK adds this automatically

Reply:
can you please change it and see if it still works?

Reply:
Run it with the debugger and indeed the 1-appid- should **not** be added. Fixed this